content/08.outlook.md

```diff
@@ -1 +1,45 @@
+Finally, mindful sharing of data with proper metadata and attribution to enable prompt data reuse is of utmost important in building datasets that can be of great value in rare disease. [@https://www.nature.com/articles/s41576-020-0257-5]
+
+
+_Development of methods that reliably support mechanistic interrogation of specific rare diseases_
+
+A pattern that emerged readily in our quest for examples of application of ML methods in rare disease is that most ML related strategies were focused on classification of a patient as a rare disease patient among other non-rare disease patients.
+We found that examples of methodologies interrogating mechanistic details of a specific rare disease were particularly lacking.
+Reasons for this include not only the lack of large datasets, but also lack of reliability or interpretability of the results from such analyses.
+We believe that an intentional push towards developing methods or analytical workflows that address this would be critical in encouraging powerful analyses using rare disease data.
+
+Method development with rare disease applications in mind requires the developers to bear the responsibility of ensuring that the resulting model is _trustworthy_.
+The field of natural language processing has a few examples of how this can be achieved.[@https://www.aclweb.org/anthology/N16-3020.pdf, @https://www.aclweb.org/anthology/P19-1073.pdf]
+One way to increase user's trust on a developed model is by helping the user understand the behavior of the developed model through providing explanations regarding why a certain model made certain predictions.[@https://www.aclweb.org/anthology/N16-3020.pdf]
+Another approach is to provide robust _error analysis_ for newly developed models to help users understand the strengths and weaknesses of a model.[@https://www.aclweb.org/anthology/P19-1073.pdf; @https://www.mitpressjournals.org/doi/abs/10.1162/COLI_a_00072; @doi:10.1093/bioinformatics/bth060]
+Adoption of these kind of approaches into biological data and analysis is still rare but is quickly becoming necessary as machine learning approaches become mainstream in biomedicine.
+
+Finally, methods that can reliably bring disparate datasets together will always remain a need of rare diseases.
+Moreover, combining data that originated from diverse modalities to create a complete picture of the disease related biology is increasingly becoming common.
+To facilitate such analyses in rare disease, methods that rely on finding structural correspondences between datasets ("anchors") may be able to transform the status-quo of using machine learning methods in rare disease.[@https://www.aclweb.org/anthology/W06-1615.pdf; @https://people.cs.umass.edu/~mahadeva/papers/IJCAI2011-DA.pdf; https://www.cell.com/cell/fulltext/S0092-8674(19)30559-8]
```

Comment:
This is a complicated topic to introduce as second to last sentence, I suggest removing or moving elsewhere

Reply:
I agree that it is a complicated topic, but it would be great to have this in the outlook. Happy to move it towards the beginning, or delve a bit deeper into this if everyone is on board.

This is based on a discussion with @jguinney, and I thought it was good food for thought (both for us and the readers) :)